packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -25,6 +25,9 @@ import {
   configureAmplify,
 } from '../cognitoUtils';
 import 'isomorphic-fetch';
+import { API } from 'aws-amplify';
```

Comment:
The current version of `aws-amplify` is `3.3.24` where as what we have in package.json is `2.x` I would prefer if we can use the latest version so to prevent any other issues that might arise with older version of the package

Reply:
Going to do it part of another PR